c2corg_ui/static/js/documentediting.js

```diff
@@ -488,6 +488,9 @@ app.DocumentEditingController.prototype.hasMissingProps = function(doc, showErro
       hasError = (!doc['lonlat'] || (doc['lonlat'][field] === null || doc['lonlat'][field] === undefined));
     } else if (field === 'date_start') {
       hasError = (doc['date_start'] === null || doc['date_start']  === undefined);
+    } else if (field === 'elevation' && doc['waypoint_type'] === 'climbing_indoor') {
```

Comment:
> because field === 'elevation' is only true if type is 'waypoints'

As for now. But who knows if a document type with an ``elevation`` attribute will not be added in the future? For instance... accident reports! https://github.com/c2corg/v6_api/blob/reports/c2corg_api/models/report.py#L35

Your assumption makes the test not robust :P 

But it's OK because ``doc['waypoint_type'] === 'climbing_indoor'`` will return ``false`` for the accident reports.